### ISSUE TITLE
Daily reports for March 23 is completely messed up. Please, return to the previous data reporting format.

### ISSUE THREAD

**Issue by rosericazondekon** [CSSEGISandData/COVID-19#1347]
Please, be consistent and report accurate data. For example, France data is moved to French Polynesia. Geographic coordinates for the Diamond cruise Ship are wrong. In addition, getting to smaller level of granularity for the US is useless and counterproductive. I suggest you report data up to the state level as you have been doing from the start. Many of us are building visualization tools on this to help various entities to solve this pandemic. We appreciate your efforts and understand that you are very busy but please, whoever is in charge of entering the data has to be consistent throughout.
**Please, return to the previous data reporting format.**

**Comment by monstertrux** [comment]
[Confirmed cases](https://github.com/CSSEGISandData/COVID-19/blob/master/csse_covid_19_data/csse_covid_19_time_series/time_series_19-covid-Confirmed.csv) is all wrong - shows no incremental cases anywhere in the last day... not possible

**Comment by ricaragao** [comment]
Exactly, you need maintain the format. Don't change, because broken many applications that use these data.

**Comment by joaolcorreia** [comment]
@CSSEGISandData  the data was already messed up, but consistently messed up, now adding columns with a different level of granularity!?. 

Revert and run the old script to process the data using the old rules (I bet on an R script no one can understand but you), if you want to add a new format **create another feed that you will have to QA but continue with the old**.

This causes less disruption to everyone who is already using the feed and lets you incrementally do whatever you want to do.

On the new structure think about NOT mixing dimensions (cities and counties and cruise ships), and NOT adding columns that represent the same thing for different time periods. 

Makes sense? Come on ... you can do it!

**Comment by dhmacq** [comment]
> [Confirmed cases](https://github.com/CSSEGISandData/COVID-19/blob/master/csse_covid_19_data/csse_covid_19_time_series/time_series_19-covid-Confirmed.csv) is all wrong - shows no incremental cases anywhere in the last day... not possible

I see this also, albeit with a few exceptions. This looks like a real error. The new global confirmed file does not have this problem.

**Comment by aravindnc** [comment]
time_series_19-covid-Confirmed.csv

time_series_19-covid-Deaths.csv

time_series_19-covid-Recovered.csv

above files seems to be no longer updating.

**Comment by ricaragao** [comment]
@aravindnc , the problem is with daily reports in this path: https://github.com/CSSEGISandData/COVID-19/tree/master/csse_covid_19_data/csse_covid_19_daily_reports

The format of day Mar-23 is different of other. This is a big problem for who automatic process files.

**Comment by aravindnc** [comment]
@ricaragao I just saw that. Thanks. 
Lucky that I was not using the day field in my dashboard.
